### PR TITLE
implement dokku report command

### PR DESCRIPTION
**Pull request by michaelshobbs**
rolls up the following commands into a single shortcut
- `uname -a`
- `free -m`
- `docker version`
- `docker -D info`
- `docker run --rm -ti gliderlabs/herokuish:latest herokuish version`
- `dokku version`
- `dokku plugin`

### CODE REVIEW

**Comment by michaelshobbs** [comment]
The wiki will need to be updated once this is merged

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -54,13 +54,7 @@ helps prioritize the most common problems and requests.
...
 etc). Please include:
 
-* The output of `uname -a`.
-* The output of `free -m`
-* The output of `docker version`.
-* The output of `docker -D info`.
-* The output of `docker run -ti gliderlabs/herokuish:latest herokuish version`
-* The output of `dokku version`.
-* The output of `dokku plugin`.
+* The output of `dokku report`.
```
Comment (josegonzalez) [inline CONTRIBUTING.md]: This is assuming everyone is on the latest dokku. Can we assume that always?

Comment (michaelshobbs) [inline CONTRIBUTING.md]: Fair point, punt on docs update until `0.5.0`?

Comment (josegonzalez) [inline CONTRIBUTING.md]: Good with me.

**Pull request merged**